breadcrumbs-ui.js

```diff
@@ -0,0 +1,51 @@
+define([
+    '$'
+], function($) {
+
+    var init = function() {
+        var $breadcrumbs = $('.c-breadcrumbs');
+
+        $breadcrumbs.each(function() {
+            var $this = $(this);
+            var $links = $this.find('.c-breadcrumbs__item');
+            var showMoreText = "Show more breadcrumbs items";
+            var $showMore = $('<li class="c-breadcrumbs__item"><button class="c-breadcrumbs__link c--show-more js-breadcrumb-show" type="button">' + showMoreText + '</button></li>');
```

Comment:
One of the things I’m recommending is a separate template for the breadcrumb item (for a different reason). It would be awesome if we could render the show more button using that same template rather than duplicating that here. @fractaltheory: any idea how hard that would be? It would mean require-ing in the dust template and calling dust.render in the UI script…